apps/backend/src/models/Fap.ts

```diff
@@ -32,7 +32,8 @@ export class FapProposal {
     public fapId: number,
     public dateAssigned: Date,
     public fapTimeAllocation: number | null,
-    public instrumentId: number,
+    public instrumentId: number | null,
+    public callId: number,
```

Comment:
Is this making FAP's tied to a specific call? Would we have to create new panels for each round?

Reply:
This is `FapProposal` model and it means that it only connects `FAP` to a `Proposal` and it uses the proposal's call id for some filtering and other needs. So the answer is no to your second question because `FAP` is not directly connected to a call but `FapProposals`